ubo_app/display.py

```diff
@@ -24,24 +25,34 @@
 from ubo_app.constants import HEIGHT, WIDTH
 
 if IS_RPI:
-    import board
-    import digitalio
-
-    cs_pin = digitalio.DigitalInOut(board.CE0)
-    dc_pin = digitalio.DigitalInOut(board.D25)
-    reset_pin = digitalio.DigitalInOut(board.D24)
-    spi = board.SPI()
-    display = ST7789(
-        spi,
-        height=HEIGHT,
-        width=WIDTH,
-        y_offset=80,
-        x_offset=0,
-        cs=cs_pin,
-        dc=dc_pin,
-        rst=reset_pin,
-        baudrate=70_000_000,
-    )
+    eeprom_data = get_eeprom_data()
+
+    if (
+        eeprom_data is None
```

Comment:
shouldn't this be the inverse (device has LCD?):
```suggestion
        eeprom_data 
        or 'lcd' in eeprom_data
        or eeprom_data['lcd'] 
        or eeprom_data['lcd']['model'] == 'st7789'
```